ansible/utility/download-image.yml

```diff
@@ -0,0 +1,21 @@
+---
+# run with --extra-vars "url=<boot iso URL>"
+- hosts:
+    kstest
+  become: true
+  become_user: root
+
+  tasks:
+
+  - name: Create directory for install images
+    file:
+      path: /home/kstest/install_images/
+      state: directory
+      owner: kstest
+      group: kstest
+
+  - name: Download new boot iso
+    get_url:
+      url: "{{ url }}"
```

Comment:
`url` is too generic. Please rename that variable to avoid collision.

Reply:
Changed to `image_url`.